openworld/sdk/generator/client/visitors/operations.py

```diff
@@ -41,7 +84,77 @@ def get_operations(parser: OpenAPIParser, model_path: Path) -> dict[str, object]
 
         sorted_operations.append(operation)
 
-    return {"operations": sorted_operations}
+    return sorted_operations
+
+
+def clean_non_schema_parameter_models(operations, non_schema_models, models_classnames_to_update):
```

Comment:
This is so hard and complicated to follow. Please refactor, split to multiple functions, and document each.